internal/mode/static/state/graph/backend_refs.go

```diff
@@ -238,12 +238,11 @@ func validateBackendTLSPolicyMatchingAllBackends(backendRefs []BackendRef) *cond
 		if referencePolicy == nil {
 			// First reference, store the policy as reference
 			referencePolicy = backendRef.BackendTLSPolicy
-		} else {
-			// Check if the policies match
-			if checkPoliciesEqual(backendRef.BackendTLSPolicy.Source, referencePolicy.Source) {
-				mismatch = true
-				break
-			}
+		} else
+		// Check if the policies match
```

Comment:
Can you move the comment so it doesn't force a weird line break in the else if statement?